### PR TITLE
Fixed group ids as per issue #33

### PR DESCRIPTION
**Pull request by sniggel**
https://github.com/spark-redshift-community/spark-redshift/issues/33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,28 +67,7 @@ This library requires Apache Spark 2.0+ and Amazon Redshift 1.0.963+.
...
-groupId: com.databricks
-artifactId: spark-redshift_2.10
-version: 3.0.0-preview1
-```
-
-**Scala 2.11**
-```
-groupId: com.databricks
-artifactId: spark-redshift_2.11
-version: 3.0.0-preview1
```
Comment (lucagiovagnoli) [inline README.md]: Should we leave this (updated with 4.0.0) ?

File: README.md [hunk 2]
```diff
@@ -67,28 +67,7 @@ This library requires Apache Spark 2.0+ and Amazon Redshift 1.0.963+.
...
-```
-
-You will also need to provide a JDBC driver that is compatible with Redshift. Amazon recommend that you use [their driver](http://docs.aws.amazon.com/redshift/latest/mgmt/configure-jdbc-connection.html), which is distributed as a JAR that is hosted on Amazon's website. This library has also been successfully tested using the Postgres JDBC driver.
```
Comment (lucagiovagnoli) [inline README.md]: Why did you move this down ?

File: README.md (hunk 1 above)
Comment (sniggel) [reply README.md]: does 4.0.0 works ? I was under the impression that the last spark-redshift was 3.0.0-preview1.

From what I understand the last release of spark-redshift community is master-SNAPSHOT only.

Thus, that is the reason why I removed it and merged this section with what was under it (snapshot).

File: README.md (hunk 2 above)
Comment (sniggel) [reply README.md]: I merged the snapshot section with the release section because there are no releases (ie. master-SNAPSHOT),

File: README.md (hunk 1 above)
Comment (lucagiovagnoli) [reply README.md]: Oh, we have 4.0.0:
https://mvnrepository.com/artifact/io.github.spark-redshift-community/spark-redshift

I meant maybe we can update that to be:
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0

File: README.md (hunk 2 above)
Comment (lucagiovagnoli) [reply README.md]: See above

File: README.md (hunk 1 above)
Comment (sniggel) [reply README.md]: Oh I didn't know 4.0.0 was there, I'll update the readme then.

File: README.md (hunk 2 above)
Comment (sniggel) [reply README.md]: yup

File: README.md (hunk 1 above)
Comment (sniggel) [reply README.md]: So I only see 2.11 for release 4.0.0 but snapshot has 2.10 and 2.11, is that accurate ?

Comment (lucagiovagnoli) [reply README.md]: Where did you see that ? 
I think SNAPSHOT has only 2.11 too:
https://oss.sonatype.org/#nexus-search;quick~spark-redshift-community

I think adding the following should be enough?
```
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0
```

Comment (sniggel) [reply README.md]: The README contains group ids that point to https://mvnrepository.com/artifact/com.github.databricks/spark-redshift
(see https://github.com/spark-redshift-community/spark-redshift/blob/master/README.md#snapshot-builds)

So that's why I was asking because I was able to use "com.github.databricks" %% "spark-redshift" % "master-SNAPSHOT" inside my sbt.

So ok, I will assume theses are valid:
```
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0
```
```
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0-SNAPSHOT
```

**Comment by lucagiovagnoli** [comment]
Thanks a lot for taking care of this! Merged.
